bin/package_all

```diff
@@ -9,40 +9,69 @@
 # Usage example:
 #> bin/package_all 0.0.1
 
+source bin/helpers/functions.sh
+source bin/helpers/load_environment.sh
+
 VERSION=$1
 if [ -z "$VERSION" ]; then
     printf "\e[0;31m%s\e[0m\n" "Missing version!"
     exit 1
 fi
 
 DIR_BUILD="build/package"
+DOCKER_IMAGE="client_builder"
+DOCKER_LOCAL_DIR="/go/src/github.com/mysterium/node"
+PLATFORMS="darwin-10.11/amd64,windows-10.0/amd64,linux/amd64"
 
 printf "Creating build directory '$DIR_BUILD' ..\n" \
     && rm -rf $DIR_BUILD \
     && mkdir -p $DIR_BUILD \
     && cp INSTALL.md $DIR_BUILD/INSTALL.txt \
     && printf "\n\n"
 
-for ARCH in amd64 i386 armhf
-do
-    bin/builder_run "bin/server_build; bin/server_package_debian $VERSION $ARCH" \
-      && mv build/server/mysterium_server $DIR_BUILD/mysterium_server_linux_$ARCH \
-      && printf "\n\n"
-done
+# build client
+docker run --rm -v $(pwd):$DOCKER_LOCAL_DIR $DOCKER_IMAGE -c "
```

Comment:
You can assume that script is already inside docker, and call this script with:
`bin/builder_run bin/package_all`.
Script ecpects environment to be ready and calls `xgo` directly without proxying to docker

Reply:
This is so confusing, we're separating files into different responsibilities, but not reusing them out of package_all script.. I would prefer single larger file than many one liners